en/docs/develop/apis/association-rest-api.md

```diff
@@ -18,13 +18,13 @@ template: templates/swagger.html
     
        1. A user named `john`, with the password `pass123`, exists in the tenant `carbon.super`. For instructions on creating a user, see [here](../../../guides/identity-lifecycles/admin-creation-workflow/).
        2. The id value for the local user association `john` is used for some API calls, which is assumed as 
-       `UFJJTUFSWS9qb2hu`(This value is returned when a GET request is made to `/me/associations`(or 
+       `UFJJTUFSWS9qb2hu` (this value is returned when a GET request is made to `/me/associations`(or 
```

Comment:
Do we need this within commas? I think making it a separate sentence gives better readability.

Reply:
made it an Info